_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,419 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+
+authors:
+  - Carlos Duarte
+---
+
+## Applicability
+
```

Comment:
According to the SC, the auto-updating information (or text, as is the case of this test rule) must be presented in parallel with other content in order to be applicable. Work this into the applicability somehow?

Also add an inapplicable test case?

Reply:
Nice catch! I'll update accordingly.